.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: node_js
 node_js:
-- stable
-script: npm run lint && npm run build
+  - stable
+before_script:
+  - npm run serve &
```

Comment:
I'm following this tutorial to set it up: https://medium.com/@mikaelberg/zero-to-hero-with-end-to-end-tests-using-nightwatch-saucelabs-and-travis-e932c8deb695